src/components/DependencyGraph.tsx

```diff
@@ -61,31 +61,31 @@ export const DependencyGraph: React.FC = () => {
     }, [showOnlyConnectedNodes, onlyNodesWithContracts, graphNetwork]);
 
     return (
-        <div className={containerCls}>
+        <>
             <div className={optionsCls}>
+                <h1 className={headerCls}>Judge-Dredd UI</h1>
                 {process.env.NODE_ENV === 'development' && (
-                    <>
-                        <Radio onChange={handleViewSwitchChange} toggle checked={showOnlyConnectedNodes} />
-
-                        <input type="range" step="1" min="1" id="dependencyLevelInput" defaultValue="1" />
-                    </>
+                    <Radio onChange={handleViewSwitchChange} toggle checked={showOnlyConnectedNodes} />
```

Comment:
``showOnlyConnectedNodes`` is not proper with JS name conventions

should be: areShownOnlyConnectedNodes or something like that